backend/src/app/entrypoints/api/routes/copilot_api.py

```
@@ -0,0 +1,153 @@
+import asyncio
+import asyncio
```

Ruff (0.11.9)

Redefinition of unused `asyncio` from line 1

Remove definition: `asyncio`

(F811)